common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -126,7 +126,7 @@
                 course_settings: this.courseSettings,
                 topicId: discussionId,
                 startHeader: this.startHeader,
-                is_commentable_cohorted: response.is_commentable_cohorted
+                is_commentable_divided: response.is_commentable_divided
```

Comment:
Small nit: Similar to AJ's comment, I know this isn't something that changed, but it just seems strange to me that there are two different forms of casing in a single file.